presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -217,6 +218,16 @@ public boolean equals(Object obj)
                     Objects.equals(this.orderBy, other.orderBy) &&
                     Objects.equals(this.orderings, other.orderings);
         }
+
+        @Override
+        public String toString()
+        {
+            return toStringHelper(this)
+                    .add("partitionBy", this.partitionBy)
+                    .add("orderBy", this.orderBy)
+                    .add("orderings", this.orderings)
```

Comment:
This is dead code because everything that prints specifications goes through toStringHelper. I could move the improved code to SpecificationProvider, but I don't think it really adds much since the actual plan is formatted using toStringHelper anyway.